<!DOCTYPE html>
<html lang="en" dir="ltr">
  <head>
    <meta charset="utf-8">
    <title>Design</title>
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet">
    <link rel="stylesheet" type="text/css" href="./style1.css" media="screen" >
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"></script>
    <script src="./loader.js"></script>
  </head>

<body>

  <nav class="navbar navbar-expand-sm navbar-dark">
  <div class="container-fluid">
    <a class="navbar-brand" href="#"><strong>BEESENSOR</strong></a>
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#collapsibleNavbar">
      <span class="navbar-toggler-icon"></span>
    </button>
    <div class="collapse navbar-collapse" id="collapsibleNavbar">
      <ul class="navbar-nav">
        <li class="nav-item">
          <a class="nav-link" href="./Description.html"style="font-size:18px;">Description</a>
        </li>
        <li class="nav-item dropdown">
          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" style="font-size:18px;">Dry Lab</a>
          <ul class="dropdown-menu">
            <li><a class="dropdown-item" href="./software.html">Software</a></li>
            <li><a class="dropdown-item" href="./drylabproject_2.html">Hardware</a></li>
            <li><a class="dropdown-item" href="./drylabproject_1.html">Kinetic Modelling</a></li>
          </ul>
        </li>
        <li class="nav-item dropdown">
          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" style="font-size:18px;">Wet Lab</a>
          <ul class="dropdown-menu">
            <li><a class="dropdown-item" href="./design.html">Design</a></li>
            <li><a class="dropdown-item" href="experimantal workflow.html">Experimental Workflow</a></li>
            <li><a class="dropdown-item" href="mock lab journal.html">Mock Lab Journal</a></li>
          </ul>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="./human practices.html"style="font-size:18px;">Human Practices</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="./attribution.html"style="font-size:18px;">Attribution</a>
        </li>

      </ul>
    </div>
  </div>
</nav>

<div class="container-banner" >
  <img src="dna.png" alt="experiment" height= 500px style="width:100%;" >
  <div class="centered-text banner-shadow" ><strong>Design</strong></div>
</div>

<div class="shadow-box">

<h1 style="color:black;margin-left:10%; margin-right:10%;">splA Plasmid</h2>

  <p style="margin-left:10%; margin-right:10%; font-size:20px">
    <br>
      <a href="https://www.uniprot.org/uniprot/I0BWH5">splA - S-layer protein A - Paenibacillus larvae</a>
    <br>
The splA S-layer protein is found to be key to the virulence of ERIC II genotypes of P. larvae, as we’ve heard from HP contacts the ERIC II genotype is not present in Alberta or North America for that matter. Thus we looked to find a protein in the ERIC I or more genotypes of P. larvae. Potential proteins that are important to the virulence strategies of P. larvae have been identified in this 2019 study of exoproteins, such as toxins Plx1 and Plx2, chitin-degrading protein PlCBP49, ADP-ribosyltransferase C3larvin toxin, and Plx2A has been shown to be a C3-like toxin. For the purposes of this wiki assignment, we have chosen to use splA due to more literature being present on it but acknowledge we would look into the proteins listed above or perhaps others to find a more appropriate protein. This protein will be secreted and purified through the use of the His-tag attached, we can then place the protein onto a matrix for aptamer-binding.
<br>
    <img src="Modified Spla Plasmid Diagram.png" style="margin-left:35%; margin-right:15%;" height=50%; width=50%; alt="table">
    <p style="margin-left:10%; margin-right:10%; font-size:20px"><em>Modified pSB1C3 plasmid for recombinant splA expression (Plasmid Diagram)</em>
    <br>
    <iframe src="Modified-splA-Plasmid Sequence Map.pdf#toolbar=0" style="margin-left:32%; margin-right:15%;" width="800px" height="500px"></iframe>
    <p style="margin-left:10%; margin-right:10%; font-size:20px"><em>Modified pSB1C3 plasmid for recombinant splA expression (Coding Sequence)</em>
<br>
</p>

    <br>

    <h1 style="color:black;margin-left:10%; margin-right:10%;">Cas12a Plasmid</h2>
      <p style="margin-left:10%; margin-right:10%; font-size:20px">

    <br>

    <a href="https://www.uniprot.org/uniprot/A0Q7Q2">CRISPR-associated endonuclease Cas12a - Francisella tularensis subsp. novicida (strain U112)</a>
        <br>
        <a href="http://parts.igem.org/Part:BBa_K2872019">Part:BBa_K2872019</a>

        <br>
    The reason we choose Cas12a instead of Cas9 is that Cas9 requires two RNA molecules: tracrRNA and a crRNA, whereas Cas12a requires only a single RNA molecule. Even though Cas9s tend to have naturally higher fidelity, Cas12 is more compact and efficient. It also processes its own guide RNAs, leading to increased multiplexing ability. This protein will be secreted and then purified through the use of the His-tag attached, there is also a protease recognition site that will allow us to snip off the His-tag to allow proper functionality of Cas12a.
    <br>
    <img src="Modified Cas12a Plasmid Diagram.png" style="margin-left:35%; margin-right:15%;" height=50%; width=50%; alt="table">
    <p style="margin-left:10%; margin-right:10%; font-size:20px"><em>Modified pSB1C3 plasmid for recombinant Cas12a expression (Plasmid Diagram)</em>
    <br>
    <iframe src="Modified-Cas12a-Plasmid Sequence Map.pdf#toolbar=0" style="margin-left:32%; margin-right:15%;" width="800px" height="500px"></iframe>
    <p style="margin-left:10%; margin-right:10%; font-size:20px"><em>Modified pSB1C3 plasmid for recombinant Cas12a expression (Coding Sequence)</em>
        <br>
        <br>
      </p>

      <h1 style="color:black;margin-left:10%; margin-right:10%;">Protein Functionality</h1>
      <p style="margin-left:10%; margin-right:10%; font-size:20px">
        <br>
        The splA S-layer protein is found to be key to the virulence of ERIC II genotypes of P. larvae, as we’ve heard from HP contacts the ERIC II genotype is not present in Alberta or North America for that matter. Thus we looked to find a protein in the ERIC I or more genotypes of P. larvae. Potential proteins that are important to the virulence strategies of P. larvae have been identified in this <a href="https://www.tandfonline.com/doi/full/10.1080/21505594.2019.1603133">2019 study</a>
        exoproteins, such as toxins Plx1 and Plx2, chitin-degrading protein PlCBP49, ADP-ribosyltransferase C3larvin toxin, and Plx2A has been shown to be a C3-like toxin. For the purposes of this wiki assignment, we have chosen to use splA due to more literature being present on it but acknowledge we would look into the proteins listed above or perhaps others to find a more appropriate protein.</p>
        <br>
        <br>

        <h1 style="color:black;margin-left:10%; margin-right:10%;">About the Protein</h1>
        <p style="margin-left:10%; margin-right:10%; font-size:20px">
          <br>
              splA has not been recombinantly expressed before but Cas12a has as it is part of the CRISPR system. Thus there is more literature on the functionality of Cas12a and more research will need to be done for splA or whichever protein gets selected. splA is a membrane bound protein as it is used by the P. larvae to infect host cells through adhesion and confirmation change.  Cas12a is not membrane bound as it is part of the CRISPR system.</p>
          <br>
          <br>

          <h1 style="color:black;margin-left:10%; margin-right:10%;">Structure</h1>
          <p style="margin-left:10%; margin-right:10%; font-size:20px">
            <br>
              The importance of the interplay between hydrophobic and hydrophilic regions were studied by reassembling the S‐layer protein SbpA on self‐assembled monolayers (SAMs) on gold composed of disulfides with different end groups [hydroxyl (OH) vs. methyl (CH3) groups] and lengths of the individual methylene chains (Moreno‐Flores et al., 2008). The formation of monolayers was observed when the hydrophobic end groups (CH3) overcame the hydrophilic (OH) ones. On the contrary, double S‐layers were formed when hydrophilic (OH) groups superseded the hydrophobic (CH3) end groups. S-layers represent highly porous protein lattices (30–70% porosity) with pores of uniform size and morphology in the 2–8 nm range.</p> <br>
            <br>

            <h1 style="color:black;margin-left:10%; margin-right:10%;">Post-Translational Modifications</h1>
            <p style="margin-left:10%; margin-right:10%; font-size:20px">
              This study highlights some of the post-translational modifications: <a href="https://pubmed.ncbi.nlm.nih.gov/11488933/">https://pubmed.ncbi.nlm.nih.gov/11488933/</a>
            </p>
              <br>
              <br>

      <h1 style="color:black;margin-left:10%; margin-right:10%;">Parts splA and Cas12a Plasmids</h1>
      <p style="margin-left:10%; margin-right:10%; font-size:20px">
        <a href="http://parts.igem.org/Part:pSB1C3">Part:pSB1C3</a>
        <br>
        pSB1C3 is a high copy number plasmid as well as being the plasmid that iGEM requires all competition submissions be made with.
      </p>

        <br>
        <br>

        <h3 style="color:black;margin-left:10%; margin-right:10%;">CmR - Chloramphenicol Resistance Gene</h3>
        <p style="margin-left:10%; margin-right:10%; font-size:20px">
          <a href="https://www.snapgene.com/resources/plasmid-files/?set=basic_cloning_vectors&plasmid=pSB1C3">pSB1C3 Sequence and Map</a>
          <br>
          CmR is a gene for resistance to the antibiotic, chloramphenicol, it is already present in the pSB1C3 plasmid.
        </p>
          <br>
          <br>

          <h3 style="color:black;margin-left:10%; margin-right:10%;">Promoter</h3>
          <p style="margin-left:10%; margin-right:10%; font-size:20px">
            <a href="http://parts.igem.org/Part:BBa_J23100">Part:BBa J23100</a>
            <br>
            This is a Constitutive Promoter compatible with E. coli. These are non-toxic proteins that are being produced in-lab for our own purposes so a Constitutive Promoter was used.
          </p>
            <br>
            <br>

            <h3 style="color:black;margin-left:10%; margin-right:10%;">RBS</h3>
            <p style="margin-left:10%; margin-right:10%; font-size:20px">
              <a href="http://parts.igem.org/Part:BBa_B0030">Part:BBa B0030</a>
              <br>
              This RBS was used by a lot of other projects.
            </p>
              <br>
              <br>

            <h3 style="color:black;margin-left:10%; margin-right:10%;">His-Tag</h3>
              <p style="margin-left:10%; margin-right:10%; font-size:20px">
                <a href="http://parts.igem.org/Part:BBa_K1223006">His-Tag</a>
                <br>
                His-tags are attached to both proteins to allow for purification.
            </p>
            <br>
            <br>

            <h3 style="color:black;margin-left:10%; margin-right:10%;">Terminator</h3>
              <p style="margin-left:10%; margin-right:10%; font-size:20px">
                <a href="http://parts.igem.org/Part:BBa_B0010">Part:BBa_B0010</a>
                <br>
                  Forward Terminator, a Bidirectional Terminator would have been chosen but the parts on the iGEM registry for some reason did not have the sequence of the parts.
            </p>
            <br>
            <br>


            <h3 style="color:black;margin-left:10%; margin-right:10%;">VF2 and VR Primer Binding Sites</h3>
              <p style="margin-left:10%; margin-right:10%; font-size:20px">
                <a href="https://www.snapgene.com/resources/plasmid-files/?set=basic_cloning_vectors&plasmid=pSB1C3">pSB1C3 Sequence and Map</a>
                <br>
                VF2 and VR are Forward and Reverse Primer Binding Sites respectively, to be used to replicate the genes of interest in the plasmid.
            </p>
            <br>
            <br>

            <h1 style="color:black;margin-left:10%; margin-right:10%;">Parts for Cas12a Plasmids</h3>
            <h3 style="color:black;margin-left:10%; margin-right:10%;">TEV Protease Recognition Site</h3>
              <p style="margin-left:10%; margin-right:10%; font-size:20px">
                <br>
                (Courtesy of Thuy-Ann Tran)
                    <br>
                This Protease Recognition Site is added to allow the His-tag to be snipped off of the Cas12a protein to allow proper function.
            </p>
            <br>
            <br>

            <h2 style="color:black;margin-left:10%; margin-right:10%;"> References:</h2>
            <p style="margin-left:10%; margin-right:10%; font-size:20px">
                Eichler, J. (2001). Post‐translational modification of the S‐layer glycoprotein occurs following translocation across the plasma membrane of the haloarchaeon Haloferax volcanii. <em>European Journal of Biochemistry, 268</em>(15), 4366-4373.
                 <br>
                 <br>
                Moreno‐Flores, S., Kasry, A., Butt, H. J., Vavilala, C., Schmittel, M., Pum, D., ... & Toca‐Herrera, J. L. (2008). From native to non‐native two‐dimensional protein lattices through underlying hydrophilic/hydrophobic nanoprotrusions. <em>Angewandte Chemie, 120</em>(25), 4785-4788.
                <br>
                <br>
                Sleytr, U. B., Schuster, B., Egelseer, E. M., & Pum, D. (2014). S-layers: principles and applications. <em>FEMS microbiology reviews, 38</em>(5), 823-864.
            </p>
</div>

</body>
</html>
